src/histogram.rs

```diff
@@ -884,8 +884,7 @@ pub fn exponential_buckets(start: f64, factor: f64, count: usize) -> Result<Vec<
 /// `duration_to_seconds` converts Duration to seconds.
 #[inline]
 pub fn duration_to_seconds(d: Duration) -> f64 {
```

Comment:
I think we can directly drop the `duration_to_seconds` function and replace it with `d.as_secs_f64()` at the calling sites.
Other than in tests, it is only used in one place, here:
https://github.com/tikv/rust-prometheus/blob/53d306635f232c4a2d9e7980da0262ec8a2c844e/src/histogram.rs#L543-L546

Reply:
Thank you for the feedback! 

Addressed the suggestion, please let me know if you think there's anything else that should be changed.